.github/workflows/changelog.yml

```diff
@@ -0,0 +1,14 @@
+name: Check Changelog
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened, labeled, unlabeled]
+    branches: [develop, main]
+jobs:
+  check-changelog:
+    name: Check Changelog Action
```

Comment:
This name doesn't seem very meaningful and does display on the check badge on the PR, but I don't have any suggestions to rename...